mmcv/runner/base_runner.py

```diff
@@ -331,11 +341,12 @@ def get_hook_info(self):
                 stage_hook_infos.append(info)
         return '\n'.join(stage_hook_infos)
 
-    def load_checkpoint(self,
-                        filename,
-                        map_location='cpu',
-                        strict=False,
-                        revise_keys=[(r'^module.', '')]):
+    def load_checkpoint(
+            self,
+            filename: str,
+            map_location: Optional[Any] = 'cpu',
+            strict: Optional[bool] = False,
```

Comment:
Optional should only be used when the default value is None, it could be better to fix the docstring.